gstcefsrc.cc

```diff
@@ -326,9 +325,23 @@ class App : public CefApp
       command_line->AppendSwitch("disable-gpu-compositing");
     }
 
-    if (src->chromium_debug_port >= 0) {
-      command_line->AppendSwitchWithValue("remote-debugging-port", g_strdup_printf ("%i", src->chromium_debug_port));
+    if (src->chrome_extra_flags) {
+        gchar ** flagsList = g_strsplit((const gchar *) src->chrome_extra_flags, ",", -1);
+        for (guint i=0 ; i<g_strv_length (flagsList) ; i++) {
```

Comment:
`for (i = 0; i < g_strv_length (flagsList); i++)` please :) (with `guint i` declared at the start of the scope)